### DISCUSSION TITLE
MassCode on brew outdated

### DISCUSSION THREAD

**Discussion by Faq** [massCodeIO/massCode#82] [General]
Hello,

Any ideas why brew still has only [v1.3.1](https://formulae.brew.sh/cask/masscode)?
Looks like they still pointing to old repository.

**Comment by panudetjt** [comment]
+1

**Comment by boldandbrad** [comment]
I'd like to see this get updated.

**Comment by Faq** [comment]
@antonreshetov ?

**Reply by antonreshetov** [reply]
@Faq Made a [PR](https://github.com/Homebrew/homebrew-cask/pull/127224) today. We are waiting. 

P.S. I did not add the previous version. That's why I was not informed about the fate of cask.

**Reply by antonreshetov** [reply]
Shipped to Homebrew 🚀

**Reply by Faq** [reply]
Thank you
